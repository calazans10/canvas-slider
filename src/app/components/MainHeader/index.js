import Header from './style';

function MainHeader() {
  return (
    <Header>
      <Header.Title>Canvas Slider</Header.Title>
    </Header>
  );
}

export default MainHeader;
