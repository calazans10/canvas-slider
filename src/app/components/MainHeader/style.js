import styled from 'styled-components';

const Header = styled.header`
  margin-bottom: 40px;
`;

Header.Title = styled.h1`
  font-size: 38px;
`;

export default Header;
