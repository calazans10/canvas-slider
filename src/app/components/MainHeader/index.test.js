import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MainHeader from '.';

describe('MainHeader', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<MainHeader />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
