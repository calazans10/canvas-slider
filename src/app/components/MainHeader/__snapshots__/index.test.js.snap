// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeader renders without crashing 1`] = `
<styled.header>
  <styled.h1>
    Canvas Slider
  </styled.h1>
</styled.header>
`;
